### ISSUE TITLE
Use of Byron's template

### ISSUE THREAD

**Issue by bsaul** [TarHeelBiostatistics/DoctoralThesisTexTemplate#14]
I found an issue in using Byron's template. His template has multiple "input" frontmatter files (e.g. titlepage.tex). I can't find a way to pass pandoc variables to input .tex files. So we have a few options:
1. Someone can figure how to pass pandoc variables to .tex files read into the main template by `\input{}`.
2. Not use Rmarkdown (not my preferred choice).
3. Put all of Byron's frontmatter files directly in the main template.tex file.

I vote for 3.

**Comment by bsaul** [comment]
Another option is just to continue develop based on the other template provided by Noorie.

**Comment by bsaul** [comment]
FWIW, the Byron template and the Noorie template took different approaches for the same problem. The Byron template took the modify individual .tex documents approach to get the desired style. The Noorie template took the create a single .tex file and a single LaTeX package that contains all the styles.

**Comment by BarkleyBG** [comment]
I may need to pull this down locally in order to get a sense of the issues going on.

If you took Option 3, would the other template still be available/usable?

**Comment by bsaul** [comment]
Why don’t I show you tomorrow. I just rolled Byron’s work into a single template file and it works pretty well.

I think we pick one template or the other so we’re not developing down two tracks.

**Comment by bsaul** [comment]
I think the new template using Byron's files as the basis is kinda slick. You can put almost everything except the main text in the YAML header:

```
---
title: "Example Dissertation"
author: 
  first: First
  last: Last
  middle: M.
date: "June 22, 2017"
fontsize: 12pt
documentclass: report
output: 
  pdf_document:
    citation_package: natbib
    keep_tex: true
    fig_caption: true
    latex_engine: pdflatex
    template: template/template2.tex
    includes: 
      in_header: example_header_include.tex
bibliography: example.bib
bibliography-style: asa
toc: true
lot: true
lof: true
committee:
  advisors: 
    - Dr. Friendly Advisor
    - Dr. Awesome Advisor
  members: 
    - Dr. Who
    - Mr. Mick Jagger
    - Dr. Strangelove
    - Dr. Doolittle
dedication: I hope we passed the audition.
abstract: Here's a little thing I wrote. You might want to read it word for word.
acknowledgements: Thanks grandma!
abbreviations:
 - short: LM
   long: "Linear Model"
 - short: GLM
   long: "Generalized Linear Model"
---
```
Then rmarkdown magic does the rest. All of these YAML variables could be easily documented. Plus, if you leave out things like the optional dedication, then the template automatically hides that page.

**Comment by BarkleyBG** [comment]
This is beautiful. Can't wait for a PR!

BTW, we've updated some gitignore stuff, so you may want to rebase or merge or whatever

**Comment by BarkleyBG** [comment]
Shall we close this now that #6 is merged?

**Comment by bsaul** [comment]
Yes. This is merge fixes #14.
